packages/browser/examples/SPARQLWidget.html

```diff
@@ -116,11 +95,27 @@
           app.getFrame3DPlanar().getLayerManager()
         );
         app.addWidgetView('layerChoice', layerChoice);
+        
+        // //// CITY OBJECTS MODULE
+        const cityObjectModule = new udvizBrowser.Widget.CityObjectModule(
+          app.getFrame3DPlanar().getLayerManager(),
+          configs['city_objects']
+        );
+        app.addWidgetView('cityObjects', cityObjectModule.view);
+
+        // TEMPORAL MODULE
+        const temporalModule = new udvizBrowser.Widget.TemporalModule(
+          app.getFrame3DPlanar().getLayerManager().tilesManagers[0],
+          configs['temporal']
+        );
+        app.addWidgetView('temporal', temporalModule.view);
 
         // //// SPARQL MODULE
         const sparqlModule = new udvizBrowser.Widget.Server.SparqlModule(
           configs['sparql_server'],
-          app.getFrame3DPlanar().getLayerManager()
+          configs['sparql_widget'],
+          app.getFrame3DPlanar().getLayerManager(),
+          cityObjectModule.provider
```

Comment:
if the sparql module take a cityobjectprovider we should do the same for the cityobjectmodule to have something symetric, in that case the user have to create a cityobjectprovider on its own to create a sparqlmodule or a cityobjectmodule. this imply in my opinion to rename the config assets/config/widget/cityobject.json to something like assets/config/style.json since this config configure more or less the stylemanager inside a tilemanager inside a layermanager inside a cityobjectprovider lol. one problem remain which is CityObjectWindow in the cityobjectmodule also need to access this config but take as parameter a cityobjectprovider lol... the quickest less dirty way to clean that would be to include the cityobjectprovider inside the layermanager and cityobjectview work with layermanager something like that but since you have better things to do i guess you could just create a cityobjectprovider with the style config and the cityobjectmodule take a cityobjectprovider (as sparqlmodule) + the style config. not sure to be clear here tell me if you need me to be more specific